typespec-tests/src/main/java/com/cadl/union/implementation/UnionClientImpl.java

```diff
@@ -88,7 +89,7 @@ public UnionFlattenOpsImpl getUnionFlattenOps() {
     /**
      * Initializes an instance of UnionClient client.
      * 
-     * @param endpoint
+     * @param endpoint Union.
```

Comment:
mark, the union added looks strange, I will check.

Reply:
Maybe OK... `@param endpoint` without a description seems not right.